### ISSUE TITLE
rtorrent.input_history with sensitive content

### ISSUE THREAD

**Issue by Peter774** [rakshasa/rtorrent#1089]
Is there a way to disable rtorrent.input_history file which contains info about downloaded torrents?

**Comment by kannibalox** [comment]
Haven't tried it myself, but symlinking it to /dev/null should work. `ln -s /dev/null <session directory>/rtorrent.input_history`.

**Comment by Peter774** [comment]
No luck.
Symlink is deleted and recreated as an regular file.

**Comment by pyroscope** [comment]
your session contains that info, too, and more. and you can rm the history file via cron every hour or so.

**Comment by George-TL** [comment]
> your session contains that info, too, and more. and you can rm the history file via cron every hour or so.

Yes, this issue is about "rtorrent.input_history" file which sits in "session.path.set" directory.

**Comment by kannibalox** [comment]
After looking at the code, one workaround I found is to create `rtorrent.input_history` as a directory. It detects that it exists but also can't properly write to it and exits silently.

**Comment by Peter774** [comment]
> After looking at the code, one workaround I found is to create `rtorrent.input_history.new` as a directory. It detects that it exists but also can't properly write to it and exits silently.

Yup, I can confirm that creating "rtorrent.input_history.new" directory and removing "rtorrent.input_history" file just works.
